BLSQ/iaso
# IA-1593: Stocks: replace the icons "settings" with an icon "edit"

Explain what problem this PR is resolving
- Stocks: replace the icons "settings" with an icon "edit"
Related JIRA tickets : [POLIO-1593](https://bluesquare.atlassian.net/browse/POLIO-1593)

## Self proofreading checklist


## Changes
- replace the icons "settings" with an icon "edit" in stock management

## How to test
- Go in Polio -> Vaccine module -> stock management -> stock variation
- Check Form A, Destruction report and incident report and check the edit ico

## Print screen / video
[Screencast from 2024-07-08 11-21-12.webm](https://github.com/BLSQ/iaso/assets/19631540/ea6bd5c3-36f9-472c-b3c2-69b974d6182a)



[POLIO-1593]: https://bluesquare.atlassian.net/browse/POLIO-1593?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
